base/collect_database.go

```diff
@@ -47,32 +45,14 @@ type Params struct {
 	Value string `json:"value" yaml:"value"`
 }
 
-const startingTimeString = "StartingTime"
-const elapsedTimeString = "ElapsedTime"
-
-// ToDo: Iter8 wiki is a great place to document the following (whichs seems IBM Code Engine specific) ... move it there
+const (
+	StartingTime = "StartingTime"
```

Comment:
`StartingTime` -- please change to `startingTime`.
`ElapsedTime` -- please change to `elapsedTime`.

***

In go, capitalizing a const or variable or func or type means you are exporting  (and exposing) it outside the package. In this case, there is no need, and these can be kept package local.